client/src/components/Template/NavBar/__tests__/NavBar.test.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { makeMockStore } from "../../../../test/testUtils";
+import { cleanup, fireEvent } from "@testing-library/react";
+import { renderWithRouter } from "../../../../test/routerTestUtils";
+import NavBar from "../NavBar";
+
+const defaultProps = {
+  auth: {
+    isAuthenticated: false,
+    user: {},
+    loading: false
+  }
+};
```

Comment:
Maybe this should be extracted for reuse?

Reply:
yup. on the other test branch, i've set up a fixture that i'm pulling in.